plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/HelmMojo.java

```diff
@@ -1,701 +0,0 @@
-/*
```

Comment:
The motivation to remove goals is not so much to reduce the number of goals, but to strip off stuff which is not really used _or_ is only half-baken. 

I think creating Helm charts might make sense really, but we need to fully support this, like updating it for the forthcoming Helm 3 release. 

> If we want to get rid of this goal, maybe we can process it at fabric8:apply time, WDYT?

I don't think that makes sense, as fabric8:helm is about creating Helm charts, which are kind of application descriptors (so it would fit better to `fabric8:resource` if so).

Reply:
If we want to get rid of helm goal, we also need to remove helm goal from here as well . https://github.com/fabric8io/fabric8-maven-plugin/blob/2e8f27c728ab4797d1ef2980873ebccbcb1436d1/samples/spring-boot/pom.xml#L119